deploy/kubernetes/console/Chart.yaml

```diff
@@ -1,8 +1,17 @@
 apiVersion: v1
-description: A Helm chart for deploying Stratos UI Console
+description: A Helm chart for deploying Stratos
 name: console
 version: 0.1.0
 appVersion: 0.1.0
 sources:
   - https://github.com/cloudfoundry/stratos
-icon: https://raw.githubusercontent.com/cloudfoundry/stratos/master/deploy/kubernetes/console/icon.png
\ No newline at end of file
+icon: https://raw.githubusercontent.com/cloudfoundry/stratos/master/deploy/kubernetes/console/icon.png
+maintainers:
+  - name: Stratos @ SUSE
+    email: "TODO: RC"
+    url: https://stratos.app
```

Comment:
This should be the 'home' field - see: https://helm.sh/docs/topics/charts/#the-chartyaml-file